e2e/content-services/permissions-component.e2e.ts

```diff
@@ -210,4 +213,50 @@ describe('Permissions Component', function () {
 
     });
 
+    describe('Role Site Dropdown', function () {
+
+        beforeEach(async (done) => {
+            await alfrescoJsApi.login(fileOwnerUser.id, fileOwnerUser.password);
+            let siteName = `PUBLIC_TEST_SITE_${Util.generateRandomString(5)}`;
+            let folderName = `MEESEEKS_${Util.generateRandomString(5)}`;
+            let publicSiteBody = { visibility: 'PUBLIC', title: siteName };
+
+            publicSite = await alfrescoJsApi.core.sitesApi.createSite(publicSiteBody);
+
+            uploadedFolder = await uploadActions.createFolder(alfrescoJsApi, folderName, publicSite.entry.guid);
+
+            loginPage.loginToContentServicesUsingUserModel(fileOwnerUser);
+            browser.get(TestConfig.adf.url + '/files/' + publicSite.entry.guid);
+            contentList.checkContentIsDisplayed(folderName);
+            contentServicesPage.checkSelectedSiteIsDisplayed('My files');
+            contentList.rightClickOnRowNamed(folderName);
+            contentList.pressContextMenuActionNamed('Permission');
+            permissionsPage.checkAddPermissionButtonIsDisplayed();
+            permissionsPage.clickAddPermissionButton();
+            permissionsPage.checkAddPermissionDialogIsDisplayed();
+            permissionsPage.checkSearchUserInputIsDisplayed();
+            permissionsPage.searchUserOrGroup(filePermissionUser.getId());
+            permissionsPage.clickUserOrGroup(filePermissionUser.getFirstName());
+            permissionsPage.checkUserOrGroupIsAdded(filePermissionUser.getId());
```

Comment:
Do we really need to repeat all those steps for every test that will be in this describe ?

Reply:
yes